aspnet/fundamentals/file-providers/sample/src/FileProviderSample/Controllers/HomeController.cs

```diff
@@ -0,0 +1,40 @@
+﻿using System;
+using Microsoft.AspNetCore.Mvc;
+using Microsoft.Extensions.FileProviders;
+
+namespace FileProviderSample.Controllers
+{
+    public class HomeController : Controller
+    {
+        private readonly IFileProvider _fileProvider;
+        private static string _dateFileLastChanged = "Never";
+
+        public HomeController(IFileProvider fileProvider)
```

Comment:
I would expect in most applications only one of them would be configured in DI. I can make that more clear, which I think is better than adding another abstraction in this case since the abstraction doesn't add anything to the File Provider content. Ok?